config.go

```diff
@@ -104,6 +104,7 @@ type ConfigNetwork struct {
 	durationControlPDU   time.Duration
 	linkbpsControl       int64
 	linkbpsData          int64
+	cmdWindowSz          int   // cmd Window size in number of chunks.
 	netdurationDataChunk time.Duration
```

Comment:
How about we introduce a convention <modelname>-config.go, for a model specific config. 
E.g., mb-config.go. We will need it, to be able to configure the latency function, for instance.
Of course, all configs must be loaded and parsed at startup.